### PR TITLE
fix(wixard.step): enable next/previous depending on selected substeps

### PR DESCRIPTION
**Pull request by corinnekrych**
Fixes https://github.com/patternfly/patternfly-ng/issues/148

### CODE REVIEW

**Comment by corinnekrych** [comment]
@dgutride @dlabrecq mind a review?

File: src/app/wizard/wizard-step.component.ts [hunk 1]
```diff
@@ -126,9 +126,10 @@ export class WizardStepComponent extends WizardBase implements OnInit, WizardSte
   get nextEnabled() {
     let enabled = this.config.nextEnabled;
     if (this.hasSubsteps) {
-      this.getEnabledSteps().forEach((step: WizardStep) => {
-        enabled = enabled && step.config.nextEnabled;
-      });
+      let selectedSubstep = this.getEnabledSteps().find(step => step.selected)
```
Comment (dgutride) [inline src/app/wizard/wizard-step.component.ts]: Please add a semicolon to end this line.

File: src/app/wizard/wizard-step.component.ts [hunk 2]
```diff
@@ -141,9 +142,10 @@ export class WizardStepComponent extends WizardBase implements OnInit, WizardSte
   get previousEnabled() {
     let enabled = this.config.previousEnabled;
     if (this.hasSubsteps) {
-      this.getEnabledSteps().forEach((step: WizardStep) => {
-        enabled = enabled && step.config.previousEnabled;
-      });
+      let selectedSubstep = this.getEnabledSteps().find(step => step.selected)
```
Comment (dgutride) [inline src/app/wizard/wizard-step.component.ts]: Semicolon missing here, too.

**Review by dgutride — CHANGES_REQUESTED** [review]
Just minor linting changes before we take it in.

**Review by dgutride — APPROVED** [review]

**Pull request merged**